backend/src/main/scala/bloop/Compiler.scala

```diff
@@ -655,10 +655,13 @@ object Compiler {
   ): Unit = {
     val label = s"Writing analysis to ${storeFile.syntax}..."
     tracer.trace(label) { _ =>
-      val filter = bloop.logging.DebugFilter.Compilation
-      logger.debug(label)(filter)
-      FileAnalysisStore.binary(storeFile.toFile).set(ConcreteAnalysisContents(analysis, setup))
-      logger.debug(s"Wrote analysis to ${storeFile.syntax}...")(filter)
+      if (analysis == Analysis.Empty || analysis.equals(Analysis.empty)) {
```

Comment:
This should have been `analysis == Analysis.Empty`, I'll fix in a follow-up commit